lib/id3c/cli/redcap.py

```diff
@@ -254,15 +254,30 @@ class Record(dict):
     Must be constructed with a REDCap :py:class:``Project``, which is stored as
     the ``project`` attribute and used to set the ``id`` attribute storing the
     record's primary id.
+
+    Note that event_name and repeat_instance might not be populated because their
+    corresponding fields (redcap_event_name and redcap_repeat_instance)
+    won't be returned by the API if the request includes the 'fields' parameter
+    but not the record_id field.
     """
     project: Project
     id: str
+    event_name: Optional[str]
+    repeat_instance: Optional[int]
```

Comment:
For completeness, it'd be good to include `repeat_instrument` too. I believe we'll even want this shortly for the childcare ingest…?

Reply:
Childcare isn't using repeated instruments, but I added repeat_instrument for future use. That's a good idea since we're in here.